FinniversKit/Sources/Recycling/GridViews/AdRecommendations/Cell/StandardAdRecommendationCell.swift

```diff
@@ -372,6 +394,15 @@ public class StandardAdRecommendationCell: UICollectionViewCell, AdRecommendatio
             performViewChanges()
         }
     }
+    
+    private func centerIconInContainer(_ shouldCenter: Bool) {
+        guard let leadingConstraint = self.iconImageLeadingConstraint,
+              let centerXConstraint = self.iconCenterXConstraint else {
+            return
+        }
+        leadingConstraint.priority = shouldCenter ? .defaultLow : .defaultHigh
```

Comment:
I have one idea. Would it work to just remove the trailing padding for the price text label if there's no price (that would be change `imageDescriptionView.trailingAnchor` spacing to 0)? Then we don't have to switch between these constraints, and the left padding would always be the same, so it looks nicer in my opinion. (now the left padding looks bigger for the centered icons, because of the bigger width).

If I'm not wrong, then the icon would look centered, because it would have `StandardAdRecommendationCell.margin` on both leading and trailing 🤔  Correct me if I'm wrong!